### ISSUE TITLE
(#1): Implement an image scraper to expand database

### ISSUE THREAD

**Issue by justintnguyen** [justintnguyen/Produce-Scanner#1]
When expanding our database, our target goal is ~5000 images for use in CNN training. Therefore, the need arises for an image scraper.

**Comment by amnguyen00** [comment]
Need to decided # of classes that will be used

**Comment by justintnguyen** [comment]
Yes, we'll need to decide the different types of fruits. Perhaps we can start off with a few (basic apples, bananas), with a preference for fruits that don't have many variations?

I'm looking into using the PIL (Python Imaging Library) for image processing. Will provide updates as I go.
